src/main/java/io/leonis/subra/gui/field/PotentialBackground.java

```diff
@@ -0,0 +1,25 @@
+package io.leonis.subra.gui.field;
+
+import com.googlecode.lanterna.*;
+import java.util.function.Function;
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+/**
+ * @author Jeroen de Jong
+ */
+public class PotentialBackground implements Function<TerminalPosition, TextColor> {
+  private final Function<Double, TextColor.RGB> gradient;
```

Comment:
Note to self: change the second generic to an interface instead of a class. `Torch` should be updated to make `Gradient` return `TextColor` first though.